env/app_vars.sh

```diff
@@ -0,0 +1,112 @@
+echo "OCW_BASE_URL=https://ocw.mit.edu/" >>.env
+echo "OCW_CONTENT_BUCKET_NAME=ocw-content-storage" >>.env
+echo "OCW_UPLOAD_IMAGE_ONLY=True" >>.env
+echo "OCW_LIVE_BUCKET=ocw-content-live-production" >>.env
+echo "OLL_ALT_URL=https://openlearninglibrary.mit.edu/courses/" >>.env
+echo "OLL_API_ACCESS_TOKEN_URL=https://openlearninglibrary.mit.edu/oauth2/access_token/" >>.env
+echo "OLL_API_URL=https://discovery.openlearninglibrary.mit.edu/api/v1/catalogs/1/courses/" >>.env
+echo "OLL_BASE_URL=https://openlearninglibrary.mit.edu/course/" >>.env
+echo "OLL_LEARNING_COURSE_BUCKET_NAME=ol-data-lake-landing-zone-production" >>.env
+echo "OLL_LEARNING_COURSE_BUCKET_PREFIX=open-learning-library/courses/" >>.env
+echo "OPENSEARCH_DEFAULT_TIMEOUT=30" >>.env
+echo "OPENSEARCH_INDEXING_CHUNK_SIZE=75" >>.env
+echo "QDRANT_COLLECTION_NAME=mitlearn-qa" >>.env
+echo "PROLEARN_CATALOG_API_URL=https://prolearn.mit.edu/graphql" >>.env
+echo "SEE_API_URL=https://mit-unified-portal-prod-78eeds.43d8q2.usa-e2.cloudhub.io/api/" >>.env
+echo "SEE_API_ACCESS_TOKEN_URL=https://mit-unified-portal-prod-78eeds.43d8q2.usa-e2.cloudhub.io/oauth/token" >>.env
+echo "SECURE_CROSS_ORIGIN_OPENER_POLICY=None" >>.env
+echo "SEE_BASE_URL=https://executive.mit.edu/" >>.env
+echo "SOCIAL_AUTH_OL_OIDC_KEY=ol-mitlearn-client" >>.env
+echo "USE_X_FORWARDED_HOST=True" >>.env
+echo "USE_X_FORWARDED_PORT=True" >>.env
+echo "XPRO_CATALOG_API_URL=https://xpro.mit.edu/api/programs/" >>.env
+echo "XPRO_COURSES_API_URL=https://xpro.mit.edu/api/courses/" >>.env
+echo "XPRO_LEARNING_COURSE_BUCKET_NAME=mitx-etl-xpro-production-mitxpro-production" >>.env
+echo "YOUTUBE_FETCH_TRANSCRIPT_SCHEDULE_SECONDS=21600" >>.env
+echo "YOUTUBE_CONFIG_URL=https://raw.githubusercontent.com/mitodl/open-video-data/mitopen/youtube/channels.yaml" >>.env
+echo "POSTHOG_ENABLED=True" >>.env
+echo "POSTHOG_TIMEOUT_MS=1000" >>.env
+echo "POSTHOG_API_HOST=https://app.posthog.com" >>.env
+echo "POSTHOG_PROJECT_ID=63497" >>.env
+echo >>.env
+echo "# Heroku Interopelated Vars" >>.env
+echo "ACCESS_TOKEN_URL=https://sso-qa.ol.mit.edu/realms/olapps/protocol/openid-connect/token" >>.env
+echo "AUTHORIZATION_URL=https://sso-qa.ol.mit.edu/realms/olapps/protocol/openid-connect/auth" >>.env
+echo 'CORS_ALLOWED_ORIGINS=["https://mitopen-rc.odl.mit.edu", "https://ocw-next.netlify.app", "https://ol-devops-ci.odl.mit.edu", "https://draft-qa.ocw.mit.edu", "https://live-qa.ocw.mit.edu"]' >>.env
+echo 'CSRF_TRUSTED_ORIGINS=["https://api.mitopen-rc.odl.mit.edu", "https://mitopen-rc.odl.mit.edu"]' >>.env
+echo "KEYCLOAK_BASE_URL=https://sso-qa.ol.mit.edu/" >>.env
+echo "MAILGUN_FROM_EMAIL=MIT Open <no-reply@discussions-mail.odl.mit.edu" >>.env
+echo "MAILGUN_SENDER_DOMAIN=discussions-mail.odl.mit.edu" >>.env
+echo "MAILGUN_URL=https://api.mailgun.net/v3/discussions-mail.odl.mit.edu" >>.env
+echo "OIDC_ENDPOINT=https://sso-qa.ol.mit.edu/realms/olapps" >>.env
+echo "SESSION_COOKIE_DOMAIN=.rc.learn.mit.edu" >>.env
```

Comment:
I think this probably needs to be `.odl.local` instead, but someone would need to test to verify.